doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -21,24 +21,24 @@ mongooseim   hard   nofile   1000000
 
 If you are using **Ubuntu**, all `/etc/pam.d/common-session*` files should include `session required pam_limits.so`.
 
-### `vm.args` and `vars.config` files
+### `vm.args` file
 
-Before making a release, some values in `rel/files/vm.args` have to be changed. 
-When using an SSL/TLS connection we advise to increase `ERL_MAX_PORTS` to `350000`. 
+This file is located in `REL_ROOT/etc/vm.args` where `REL_ROOT` is the path
```

Comment:
This file is located in `REL_ROOT/etc/vm.args` where `REL_ROOT` is the path to a MonoogseIM release (ie. `_build/prod/rel/mongooseim` if you build MongooseIM from source).